packages/web-components/src/components/va-breadcrumbs/va-breadcrumbs.tsx

```diff
@@ -81,11 +115,24 @@ export class VaBreadcrumbs {
   }
 
   private handleListNode(node: HTMLSlotElement, index: number, slotNodes: Node[]) {
-    node.classList.add('va-breadcrumbs-li');
+    if (this.uswds) {
+      node.classList.add('usa-breadcrumb__list-item');
+    } else {
+      node.classList.add('va-breadcrumbs-li');
+    }
     const anchor = node.querySelector('a');
     if (anchor && index === slotNodes.length - 1) {
       /* eslint-disable-next-line i18next/no-literal-string */
       anchor.setAttribute('aria-current', 'page');
+      if (this.uswds) {
+        // Replace the anchor with a span for the last list item.
+        const span = document.createElement('span');
+        span.textContent = anchor.textContent;
+        node.classList.add('usa-current');
+        node.replaceChild(span, anchor);
+        /* eslint-disable-next-line i18next/no-literal-string */
+        node?.setAttribute('aria-current', 'page');
```

Comment:
This doesn't appear to be working, didn't see this attribute in Chromatic. Maybe try moving it up before the replaceChild call?

Reply:
I think it was not needed so I added a conditional attribute to fix it and removed that code.